x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/convert/ToLongTests.java

```diff
@@ -230,7 +232,7 @@ public static Iterable<Object[]> parameters() {
         return parameterSuppliersFromTypedDataWithDefaultChecks(
             true,
             suppliers,
-            (v, p) -> "boolean or counter_integer or counter_long or datetime or numeric or string"
+            (v, p) -> "boolean or counter_integer or counter_long or date_nanos or datetime or numeric or string"
```

Comment:
Should I look into making sure these don't leak `date_nanos` when the feature flag is disabled?